client/state/data-layer/wpcom/sites/automated-transfer/eligibility/index.js

```diff
@@ -56,7 +56,7 @@ export const eligibilityHoldsFromApi = ( { errors = [] }, options = {} ) =>
 			}
 			return get( statusMapping, code, '' );
 		} )
-		.filter( identity );
+		.filter( ( error ) => error );
```

Comment:
This is a way to write `_.compact`, isn't it? A common idiom is to write `.filter( Boolean )`. That would also work for the post excerpt `.find( content => content.length )` code.

Reply:
Agreed 👍  Done in https://github.com/Automattic/wp-calypso/pull/52383/commits/8329ed34d42cf843f8f8be887ff27ef21106bf83.